dart-sdk/rust/src/api/client.rs

```diff
@@ -32,20 +32,31 @@ impl CoreClient {
         base_url: Option<String>,
         #[frb(default = "30000")] poll_interval_ms: u64,
         #[frb(default = "3000")] poll_jitter_ms: u64,
+        #[frb(default = "info")] log_level: String,
     ) -> CoreClient {
-        log::set_max_level(log::LevelFilter::Info);
+        log::set_max_level(log::LevelFilter::Debug);
 
-        let base_url = base_url.unwrap_or_else(|| DEFAULT_BASE_URL.to_owned());
+        {
+            let mut builder = env_logger::Builder::from_env(
```

Comment:
We have another logger setup from via default utils: https://github.com/Eppo-exp/eppo-multiplatform/blob/b6ede436e1a9ea9657d7f2fc158328d9c3af59fc/dart-sdk/rust/src/api/mod.rs#L61

From what I get, it integrates better with whatever platform is using for logging. And env logger will likely conflict with it.

I think we just need to pass the desired level to `set_max_level` and that should be enough?